woocommerce/woocommerce-android
# Store creation: Plans screen design update & crash fix

This PR fixes a crash caused by lambdas and update the UI based on the design changes.

<img width="300" src="https://user-images.githubusercontent.com/1522856/202671343-5073383c-3aef-41ab-842c-b520076fa9a9.png" />

**To test:**
Make sure the design matches the Figma design. 

To test the crash, enable `Don't keep activities`, then leave the app and come back and verify it doesn't crash.
